lib/tasks/tarball.rake

```diff
@@ -88,8 +88,13 @@ task :tarball do
   config = Packaging::Configuration.instance
   target = File.join(config.package_dir, package_file_name)
   begin
-    Rake::Task[target].invoke
-    build_tarball
+    puts "* Making a tarball..." if verbose
```

Comment:
NP: Maybe we could print the tarball name (stored in `target`) so you know where to look.

```suggestion
    puts "* Making tarball #{target} ..." if verbose
```

(Not tested!)